src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_archive_utils.py

```diff
@@ -208,8 +209,15 @@ def check_remote_source_code(source_location):
     # http
     if lower_source_location.startswith("https://") or lower_source_location.startswith("http://") \
        or lower_source_location.startswith("github.com/"):
-        if re.search(r"\.git(?:#.+)?$", lower_source_location) or "visualstudio.com" in lower_source_location:
-            # git url must contain ".git" or be from VSTS
+        isVSTS = False
+        for vstsURL in TASK_VALID_VSTS_URLS:
+            if vstsURL in lower_source_location:
+                isVSTS = True
+                break
```

Comment:
`isVSTS = any(url in lower_source_location for url in task_valid_urls)`

Reply:
Thanks, good one liner.